src/main/java/org/candlepin/controller/util/EmptyContentPrefix.java

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2009 - 2021 Red Hat, Inc.
+ *
+ * This software is licensed to you under the GNU General Public License,
+ * version 2 (GPLv2). There is NO WARRANTY for this software, express or
+ * implied, including the implied warranties of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. You should have received a copy of GPLv2
+ * along with this software; if not, see
+ * http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
+ *
+ * Red Hat trademarks are not licensed under GPLv2. No permission is
+ * granted to use or replicate Red Hat trademarks that are incorporated
+ * in this software or its documentation.
+ */
+
+package org.candlepin.controller.util;
+
+/**
+ * An empty prefix used for generation of Ueber certificates
+ */
+public class EmptyContentPrefix implements ContentPrefix {
```

Comment:
If ContextPrefix is changed to be a functional interface, we can omit this entirely with a simple lambda, which is actually shorter than this class name: `envId -> ""`

Reply:
Done.